scripts/build_final_apk.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+cd $(git rev-parse --show-toplevel)
+
+./scripts/rust_build_yarn_install.sh
+yarn
```

Comment:
why yarn install again?
i thought the rust_build_yarn_install.sh script handles this?

Reply:
@Oscar-Pepper you are right with this... the script don't need to run yarn twice.